var/spack/repos/builtin/packages/upcxx/package.py

```diff
@@ -150,8 +159,12 @@ def install(self, spec, prefix):
             env['CC'] = real_cc
             env['CXX'] = real_cxx
 
-            installsh = Executable("./configure")
-            installsh('--prefix=' + prefix)
+            options = ["--prefix=%s" % prefix]
+
+            if '+gasnet' in self.spec:
+                options.append('--with-gasnet=%s/src' % spec['gasnet'].prefix)
```

Comment:
This could be `options.append('--with-gasnet=%s' % spec['gasnet'].prefix.src)`

Reply:
Fixed as suggested in 2f2f6b6

Reply:
Looks like the `cae743d` undid that fix.

Reply:
oops - no I accidentally just fixed only one of two instances of the same pattern.

Second instance fixed in d40594b